pages/common/pip-hash.md

```diff
@@ -0,0 +1,20 @@
+# pip hash
+
+> Compute hashes of package archives for verification.
+> More information: <https://pip.pypa.io/en/stable/cli/pip_hash/>.
+
+- Generate hash for a package file:
+
+`pip hash {{path/to/package.whl}}`
+
+- Generate hash using a specific algorithm:
+
+`pip hash {{[-a|--algorithm]}} {{sha256}} {{path/to/package.whl}}`
```

Comment:
```suggestion
`pip hash {{[-a|--algorithm]}} {{sha256|sha384|sha512|...}} {{path/to/package.whl}}`
```
This would condense the previously redundant commands to one.